Assets/Plugins/CountlySDK/Models/CountlyConfiguration.cs

```diff
@@ -58,5 +64,36 @@ public override string ToString()
         {
             return $"{nameof(Salt)}: {Salt}, {nameof(EnablePost)}: {EnablePost}, {nameof(EnableConsoleLogging)}: {EnableConsoleLogging}, {nameof(IgnoreSessionCooldown)}: {IgnoreSessionCooldown}, {nameof(NotificationMode)}: {NotificationMode}, {nameof(EnableManualSessionHandling)}: {EnableManualSessionHandling}, {nameof(SessionDuration)}: {SessionDuration}, {nameof(EventQueueThreshold)}: {EventQueueThreshold}, {nameof(StoredRequestLimit)}: {StoredRequestLimit}, {nameof(TotalBreadcrumbsAllowed)}: {TotalBreadcrumbsAllowed}, {nameof(EnableAutomaticCrashReporting)}: {EnableAutomaticCrashReporting}";
         }
+
+        /// <summary>
+        /// Disabled the location tracking on the Countly server
+        /// </summary>
+        public void DisableLocation()
+        {
+            IsLocationDisabled = true;
+
+            City = null;
+            IPAddress = null;
+            CountryCode = null;
+            Location = string.Empty;
+        }
+
+        /// <summary>
+        ///     Set location parameters that will be used during init.
+        /// </summary>
+        /// <returns></returns>
+        public void SetLocation(string countryCode, string city, string gpsCoordinates, string ipAddress)
+        {
+            if (countryCode != null || city != null || gpsCoordinates != null || ipAddress != null)
+            {
+                City = city;
+                IPAddress = ipAddress;
+                CountryCode = countryCode;
+                Location = gpsCoordinates;
+
+                IsLocationDisabled = false;
```

Comment:
Keep the config file dumb and move the logic to the SDK. Remove this value clearer.

Reply:
done.